src/Itinerary/Contexts/ICT003Plans/Plans.tsx

```diff
@@ -60,11 +60,7 @@ export const ICT003PlansProvider = ({
     }, [parentDocRef]);
 
     const create: ICT003PlansValInterface['create'] = async () => {
-        addDoc<ICT003PlansInterface>(collectionRef, ICT003PlansBuild());
-    }
-
-    if (!documentSnapshots) {
-        return <ActivityIndicator animating={true} />
+        return await addDoc<ICT003PlansInterface>(collectionRef, ICT003PlansBuild());
```

Comment:
```suggestion
        return await create();
```
@Ayato-kosaka 
create()で問題なさそうだけど、ここで上のcreate動かすのは、邪道？

Reply:
@Takapy1 
ん？ここはcreate()の中だよ？

Reply:
修正箇所だった😂

Reply:
@Ayato-kosaka 
仕組み化、保守性がすごいな

Reply:
ね笑笑